website/docs/language/tests/mocking.mdx

```diff
@@ -127,7 +137,7 @@ mock_provider "aws" {
 }
 ```
 
-In the above example, Terraform uses the supplied value for `arn` attributes in S3 buckets instead of generating a random string. Computed attributes not provided an explicit default will simply fall back to the generic data generation rules.
+In the above example, Terraform uses the supplied value for `arn` attributes in S3 buckets instead of generating a random string. Computed attributes not provided an explicit default will simply fall back to the generic data generation rules. The `override_during` attribute can also be used in the `mock_resource` and `mock_data` blocks to specify when the values should be generated at an individual resource level. If not specified, the values will be generated using the rules inherited from the `mock_provider` block. If specified, the local value will override any value specified in the `mock_provider` block.
```

Comment:
```suggestion
In the above example, Terraform uses the supplied value for `arn` attributes in S3 buckets instead of generating a random string. Computed attributes not provided an explicit default will simply fall back to the generic data generation rules. You can also use the `override_during` attribute in the `mock_resource` and `mock_data` blocks to specify when Terraform should generate the values for an individual resource. If you do not not specify the `override_during` attribute, Terraform generates the values using the rules inherited from the `mock_provider` block. If specified, the local value overrides any value specified in the `mock_provider` block.
```

Reply:
Do we want to include an example of the `override_during` attribute?

Reply:
We do provide an example of using it in the `mock_provider` block, and the behaviour is the same across the three blocks so I didn't think it was needed - but I'm happy to be corrected.

Reply:
Oh you're totally right! Sorry, I missed that. I think this is good as-is then.